test.config.mac

```diff
@@ -0,0 +1,32 @@
+## ----------------------------------------------------------------------------
```

Comment:
These two files should also be removed by the PR, right?

Reply:
Removed!